spec/migrations/20181119154009_set_vm_connection_state_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_migration
+
+describe SetVmConnectionState do
+  let(:vm_stub)          { migration_stub(:VmOrTemplate) }
+  let!(:vm)              { vm_stub.create!(:id => id, :connection_state => connection_state) }
+  let(:region)           { vm_stub.my_region_number }
+  let(:connection_state) { nil }
+  let(:id)               { vm_stub.id_in_region(123, region) }
+
+  migration_context :up do
+    describe 'in my region' do
+      context 'connection_state is nil' do
+        it 'flips to connected' do
+          migrate
+          vm.reload
+          expect(vm.connection_state).to eq('connected')
+        end
+      end
+
+      context 'vm is disconnected' do
+        let(:connection_state) { 'disconnected' }
+        it 'disconnected stays disconnected' do
+          migrate
+          vm.reload
+          expect(vm.connection_state).to eq(connection_state)
+        end
+      end
+    end
+
+    describe 'in other region' do
```

Comment:
@miha-plesko we don't need to test that `in_my_region` works so you can just get rid of this and simplify the tests a bit